spec/models/organization_spec.rb

```diff
@@ -270,7 +270,11 @@
 
   describe "#by_customers(categories_type, :income)" do
     let(:org) { create :organization, default_currency: 'USD' }
-    let!(:zero_transaction) { create :transaction, :with_customer, :income, bank_account: account, amount: 0 }
+    let!(:zero_transaction) do
+      tr = build :transaction, :with_customer, :income, bank_account: account, amount: 0
```

Comment:
@dissident you may write instead `build(:transaction, ...).tap{ |tr| tr.save(...) }`